python/lsst/ts/ess/csc/data_client/controller_data_client.py

```diff
@@ -329,18 +294,22 @@ async def run_command(self, command: str, **parameters: Any) -> None:
             If it takes more than COMMUNICATE_TIMEOUT seconds
             to acquire the lock or write the data.
         """
-        json_str = json.dumps({"command": command, "parameters": parameters})
+        data = {
+            common.JsonKeys.COMMAND: command,
+            common.JsonKeys.PARAMETERS: parameters,
+        }
```

Comment:
If you do anything to make this easier in ts_ess_common I hope you will use it here. Not that it's a problem here (it is nicely encapsulated). I am just noting it because this same bit ended up being needed in several places in ts_ess_common.

Reply:
`JsonKeys` was removed in ts_ess_common and these lines were updated accordingly.